lib/__tests__/node/prepare.js

```diff
@@ -142,4 +145,81 @@ describe('prepare', () => {
     // received a second request it didn't expect.
     should(storeState.isRejected()).be.exactly(false, storeState.reason);
   });
+
+  describe('preparedApp', () => {
+    it('is rendered as the original app', async () => {
+      function Foo() {
+        return <span>Foo</span>;
+      }
+      const app = <Foo />;
+      const preparedApp = await prepare(app);
+      const appHtml = ReactDOMServer.renderToString(app);
+      const preparedAppHtml = ReactDOMServer.renderToString(preparedApp);
+      should(preparedAppHtml).be.exactly(appHtml);
+    });
+    it('is rendered as the original app with nested elements', async () => {
+      function Bar() {
+        return <span>Bar</span>;
+      }
+      function Foo() {
+        return <Bar />;
+      }
+      const app = <Foo />;
+      const preparedApp = await prepare(app);
+      const appHtml = ReactDOMServer.renderToString(app);
+      const preparedAppHtml = ReactDOMServer.renderToString(preparedApp);
+      should(preparedAppHtml).be.exactly(appHtml);
+    });
+    it('is rendered as the original app with null children', async () => {
+      function Bar() {
+        return null;
+      }
+      function Foo() {
+        return (
+          <div>
+            Foo
+            <Bar />
+          </div>
+        );
+      }
+      const app = <Foo />;
+      const preparedApp = await prepare(app);
+      const appHtml = ReactDOMServer.renderToString(app);
+      const preparedAppHtml = ReactDOMServer.renderToString(preparedApp);
+      should(preparedAppHtml).be.exactly(appHtml);
+    });
+    it('is rendered as the original app with false children', async () => {
+        function Bar() {
+            return false;
+        }
+        function Foo() {
+          return (
+            <div>
+              Foo
+              <Bar />
+            </div>
+          );
+        }
+        const app = <Foo />;
+        const preparedApp = await prepare(app);
+        const appHtml = ReactDOMServer.renderToString(app);
+        const preparedAppHtml = ReactDOMServer.renderToString(preparedApp);
+        should(preparedAppHtml).be.exactly(appHtml);
+    });
+    it('is rendered as the original app with special elements', async () => {
+      function Foo() {
+        return (
+          <div>
+            Foo
+            <input />
```

Comment:
Does it work well with emtpy element with closing tags ? e.g. `<div></div>` or perhaps `<div>{null}</div>`.